reference/lidar.md

```diff
@@ -149,6 +201,75 @@ void wb_lidar_disable(WbDeviceTag tag);
 int wb_lidar_get_sampling_period(WbDeviceTag tag);
 ```
 
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Lidar.hpp>
+
+namespace webots {
+  class Lidar : public Device {
+    virtual void enable(int sampling_period);
+    virtual void disable();
+    int getSamplingPeriod() const;
+    // ...
+  }
+}
+```
+
+%tab-end
+
+%tab "Python"
+
+```python
+from controller import Lidar
+
+class Lidar (Device):
+    def enable(self, sampling_period):
+    def disable(self):
+    def getSamplingPeriod(self):
+    # ...
+```
+
+%tab-end
+
+%tab "Java"
+
+```java
+import com.cyberbotics.webots.controller.Lidar;
+
+public class Lidar extends Device {
+  public void enable(int sampling_period);
+  public void disable();
+  public int getSamplingPeriod();
+  // ...
+}
+```
+
+%tab-end
+
+%tab "MATLAB"
+
+```matlab
+wb_lidar_enable(tag, sampling_period)
+wb_lidar_disable(tag)
+period = wb_lidar_get_sampling_period(tag)
+```
+
+%tab-end
+
+%tab "ROS"
+
+| name | service/topic | data type | data type definition |
+| --- | --- | --- | --- |
+| `/<device_name>/enable` | `service` | [`webots_ros::set_int`](ros-api.md#common-services) | |
+| `/<device_name>/get_sampling_period` | `service` | [`webots_ros::get_int`](ros-api.md#common-services) | |
```

Comment:
Generally, in the ROS table, the hyperlinks are not visible unless you move the mouse over them and see that you can click them. I believe we should make them more visible (using a CSS rule) otherwise, users will miss them.